reactive/ntpmon.py

```diff
@@ -17,19 +18,36 @@
     when_not,
 )
 
+import charmhelpers.contrib.templating.jinja as templating
 import os
 import subprocess
+import sys
 
 
 #
-# Module configuration - these shouldn't need adjusting
+# Module configuration
 #
 
+chrony_conf = '/etc/chrony/chrony.conf'
 ntp_conf = '/etc/ntp.conf'
-ntpmon_dir = '/opt/ntpmon'
-service_name = 'ntpmon-telegraf'
-systemd_config = '/etc/systemd/system/' + service_name + '.service'
-upstart_config = '/etc/init/' + service_name + '.conf'
+ntpmon_options = layer.options('ntpmon')
```

Comment:
PEP8 prefers module level constants (ntp_conf, chrony_conf. ntpmon_options) to be UPPER_CASE_WITH_UNDERSCORES.

Reply:
Updated